packages/mailer/src/GenericEmail.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Tempest\Mail;
+
+use Tempest\View\View;
+
+/**
+ * Represents a generic email.
+ */
+final class GenericEmail implements Email
+{
+    public Envelope $envelope {
+        get => new Envelope(
+            subject: $this->subject,
+            to: $this->to,
+            from: $this->from,
+            cc: $this->cc,
+            bcc: $this->bcc,
+            replyTo: $this->replyTo,
+            headers: $this->headers,
+            priority: $this->priority,
+        );
+    }
+
+    public Content $content {
+        get => new Content(
+            html: $this->html,
+            text: $this->text,
+            attachments: $this->attachments,
+        );
+    }
+
+    public function __construct(
+        public ?string $subject,
+        public null|string|array|Address $to,
```

Comment:
I'm not a fan of the constructor on `GenericEmail` now accepting standalone parameters and then wrapping them in `Envelope`. I'd say just have people provide `Envelope`. (Assuming `Content` goes away)